src/js/user-profile/components/FormItem.jsx

```diff
@@ -40,8 +44,11 @@ class FormItem extends React.Component {
           <i className="fa fa-close" aria-label="Close icon"></i>
```

Comment:
It seems like we shouldn't have a close icon here for now? It looks like it will delete the application, but that seems confusing.

Reply:
@stacyhlinn Could you please share your thoughts on whether the close icon and remove function should be removed?